api/helpers.go

```diff
@@ -4,22 +4,13 @@ import (
 	"encoding/json"
 	"net/http"
 
-	jwt "github.com/dgrijalva/jwt-go"
-	"github.com/netlify/gocommerce/claims"
+	"github.com/Sirupsen/logrus"
 )
 
 func sendJSON(w http.ResponseWriter, status int, obj interface{}) {
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(status)
-	encoder := json.NewEncoder(w)
-	encoder.Encode(obj)
-}
-
-func userIDFromToken(token *jwt.Token) string {
-	if token == nil {
-		return ""
+	if err := json.NewEncoder(w).Encode(obj); err != nil {
```

Comment:
So this is a weird situation. If any writes happen to `http.ResponseWriter` without there being a status header yet, it will write `http.StatusOK`. Thus, the encode cannot go before the `WriteHeader` call. We could replace with a `Marshal` call instead, but I'm guessing the performance is not as good? I'd have to look at the implementation of using an `Encoder` vs `Marshal`.